src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,23 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int LETTERS_LIMIT = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> characters;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            characters = reader.lines()
+                    .flatMapToInt(String::chars)
+                    .mapToObj(i -> (char) i)
+                    .filter(Character::isAlphabetic)
```

Comment:
```suggestion
                    .filter(Character::isLetter)
```
see the difference between these two methods)

Reply:
I agree, I was just trying to meet the requirements here (Use StringBuilder and використовуйте TreeSet). As for me better way for solving this task is something like this:
`    public String getUniqueCharactersTwo(String fileName) throws IOException {
        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
            return reader.lines()
                    .flatMapToInt(String::chars)
                    .mapToObj(i -> (char) i)
                    .filter(Character::isLetter)
                    .map(Character::toLowerCase)
                    .distinct()
                    .sorted()
                    .limit(LETTERS_LIMIT)
                    .map(String::valueOf)
                    .collect(Collectors.joining());
        }
    }`
But there is no TreeSet and StringBuilder.